signatures/helpers/helpers.go

```diff
@@ -173,3 +176,533 @@ func GetPathFromRawAddr(addr map[string]string) (string, error) {
 
 	return path, nil
 }
+
+func GetTraceeStringByName(event trace.Event, argName string) (string, error) {
```

Comment:
Shouldn't this be named GetArgumentStringByName?

Reply:
Actually this isn't needed. In `arguments_helpers.go` there is already a `GetTraceeStringArgumentByName()`.

Reply:
This isn't needed actually. There is a function called `GetTraceeStringArgumentByName` in `signatures/helpers/arguments_helpers.go`.

Reply:
Fixing...